README.md

```diff
@@ -17,6 +19,26 @@ There is a [Compose file](minimal-distributed.yaml) to instantiate a minimal (no
     # Build all the other Clearwater Docker images and start a deployment.
     docker-compose -f minimal-distributed.yaml up
 
+## Manual Turn-Up
+
+If you can't or don't want to use Compose, you can turn the deployment up manually under Docker.  To use it, run
+
+    # Install Docker (on Ubuntu) - we need the latest for compatibility with Compose.
```

Comment:
We don't use compose in this mode... do we still need the latest?  At the least, drop the compose comment.